packages/gasket-plugin-nextjs/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # `@gasket/plugin-nextjs`
 
+### 6.45.3
```

Comment:
Could you remove this line? The gasket team will establish the release version numbers when they publish.

Reply:
Done.